frontend/packages/container-security/src/components/image-vulnerability-utils.ts

```diff
@@ -0,0 +1,30 @@
+import { Vulnerability } from '../types';
+
+export enum VulnerabilitiesType {
+  allVulnerabilities = 'All vulnerabilities',
+  appDependency = 'App dependency',
+  baseImage = 'Base image',
+}
+
+const vulnerabilityMetadata = (vulnerability: Vulnerability) => {
+  return vulnerability.metadata ? JSON.parse(vulnerability.metadata) : vulnerability.metadata;
+};
```

Comment:
`JSON.parse` parse should be in `try catch` block and this doesn't seem right, should this be

```suggestion
const vulnerabilityMetadata = (vulnerability: Vulnerability) => {
  if(!vulnerability?.metadata) return;
  let metaDataVal;
  try {
     metaDataVal = JSON.parse(vulnerability.metadata)
  } catch {
     metaDataVal = vulnerability.metadata;
  }
  return metaDataVal;
};
```